a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -131,6 +136,21 @@ def get_delete_member_flow(self):
 
         return delete_member_flow
 
+    def _get_delete_member_vlan_network_subflow(self):
```

Comment:
Don't need a subflow for this. This can be compressed into a `DeleteEthernetTagIfNotInUse` task

Reply:
Done.